### DISCUSSION TITLE
Catch exceptions to prevent CefSharp from crashing the app for clients?

### DISCUSSION THREAD

**Discussion by h5kk** [cefsharp/CefSharp#4663] [General]
Hi, 

We're distributing a client application and unfortunately one of our installation methods does not support also installing the dependencies in rare situations. 

The goal here is to:

- [ ] Catch or prevent the exception so our application does not crash (cefsharp portion is not essential to functionality)
- [ ] Prompt the user to install the missing dependency (or install it for them if possible)

Thank you very much for the help.

**Comment by amaitland** [comment]
> does not support also installing the dependencies in rare situations.

What dependencies exactly? VC++?

> Catch or prevent the exception so our application does not crash (cefsharp portion is not essential to functionality)

What is the exception you are seeing?

**Reply by amaitland** [reply]
For reference https://learn.microsoft.com/en-us/cpp/windows/redistributing-visual-cpp-files?view=msvc-170#install-individual-redistributable-files
